stdlib/messaging/rabbitmq/src/main/ballerina/rabbitmq/channel_listener.bal

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Public Ballerina API - Ballerina RabbitMQ Message Listener.
+# To provide a listener to consume messages from RabbitMQ.
+#
+# + chann - Reference to a Ballerina RabbitMQ Channel.
+public type ChannelListener object {
+
+    *AbstractListener;
+
+    private Channel? chann;
```

Comment:
Shall we use a proper name for this?

Reply:
I tried using channel, but as it is a key word in ballerina i left it as chann